src/uproot_browser/tui/browser.py

```diff
@@ -124,11 +133,9 @@ def action_quit_with_dump(self) -> None:
     def action_toggle_theme(self) -> None:
         """An action to toggle dark mode."""
         self.dark = not self.dark
-        content_switcher = self.query_one("#main-view", textual.widgets.ContentSwitcher)
```

Comment:
FYI, I think this is a bug in textual - query_one doesn’t seem to work if the screen is not the active screen, but is in the background. Saving the widget where we can access it works as a workaround (and is pretty nice).